### PR TITLE
Add test case for web socket transport

### PR DESCRIPTION
**Pull request by nuwanw**
1) Start a Web Socket Sample Server
2) Configure EI with Web Socket and deploy inbound endpoint
3) Send a web socket request to the EI inbound endpoint and verify the response

### CODE REVIEW

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/websocket/inbound/transport/test/WebSocketInboundTestCase.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/*
+*Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (vinok88) [inline integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/websocket/inbound/transport/test/WebSocketInboundTestCase.java]: correct the licence year.

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/websocket/inbound/transport/test/WebSocketInboundTestCase.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.esb.websocket.client.WebSocketTestClient;
+import org.wso2.carbon.esb.websocket.server.WebSocketServer;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+public class WebSocketInboundTestCase extends ESBIntegrationTest {
```
Comment (vinok88) [inline integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/websocket/inbound/transport/test/WebSocketInboundTestCase.java]: Shall we add a class commebt.

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/websocket/inbound/transport/test/WebSocketInboundTestCase.java (hunk 1 above)
Comment (nuwanw) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/websocket/inbound/transport/test/WebSocketInboundTestCase.java]: Fixed

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/websocket/inbound/transport/test/WebSocketInboundTestCase.java (hunk 2 above)
Comment (nuwanw) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/websocket/inbound/transport/test/WebSocketInboundTestCase.java]: Added the doc comment

**Pull request merged**